libraries/lygia.yaml

```diff
@@ -0,0 +1,14 @@
+name: lygia
+description: |
+  LYGIA is a shader library of reusable functions that will let you prototype, port or ship a project in just few minutes. It's very granular, flexible and efficient. Support multiple shading languages and can easily be added to any project, enviroment or framework of your choice. 
+category: drawing, color, algorithms, 3d, animation
```

Comment:
Oh also, currently we're going to be rendering the page as a list of libraries grouped by category rather than using these as tags. Is there one of these that you think Lygia fits into best? It's a tough one because it does so much

Reply:
oh this is a tough one. Can we create one like: GL, GPU, computer graphics?

Reply:
How would you feel about renaming `filters` to `shaders`? I was initially trying to make a category that was a bit more general sounding, but it feels like other tools like Shader Park might also fit well into something like that.

Reply:
that will fit way better